src/http.c

```diff
@@ -684,8 +738,9 @@ int auth_log_in(struct tunnel *tunnel)
 
 		url_encode(tokenresponse, cfg->otp);
 		snprintf(data, sizeof(data),
-		         "username=%s&realm=%s&reqid=%s&polid=%s&grp=%s&code=%s&code2=&redir=%%2Fremote%%2Findex&just_logged_in=1",
-		         username, realm, reqid, polid, group, tokenresponse);
+		         "username=%s&realm=%s&reqid=%s&polid=%s&grp=%s&code=%s&code2=&portal=%s&peer=%s&magic=%s",
```

Comment:
Yes. The portal, peer and magic variables are needed. Again, you can keep the extra redir options if you want them to stay. 
The portal option is needed more generally (not just host checks) if the Forti is configured for multi-portal. Without that, openforti will always connect to the default portal.

Reply:
Perfect then, thank you for the explanations. No need to keep the extra `redir` if it's not required. I was worried it might be needed by older Fortigate appliances or specific use cases but it looks it should be removed.